openedx/features/caliper_tracking/tests/expected/xblock.poll.view_results.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Viewed",
```

Comment:
Is there a better option instead of `Viewed`. Because `Viewed` is deprecated in `AssessmentItemEvent`

Reply:
Good catch.

Reply:
Main type of event is changed from "AssessmentItemEvent" to "Event"